pkg/api/playlist.go

```diff
@@ -92,20 +84,20 @@ func (hs *HTTPServer) LoadPlaylistItemDTOs(ctx context.Context, id int64) ([]mod
 
 	for _, item := range playlistitems {
 		playlistDTOs = append(playlistDTOs, models.PlaylistItemDTO{
-			Id:         item.Id,
-			PlaylistId: item.PlaylistId,
-			Type:       item.Type,
-			Value:      item.Value,
-			Order:      item.Order,
-			Title:      item.Title,
+			Id:          item.Id,
```

Comment:
I think that it does not harm leaving it there since it can help for debugging issues (cross-checking with the database contents)
Also, I don't think that we plan to remove the `id` column from the table.